src/lib/containers/home_page/programs/Programs.tsx

```diff
@@ -78,17 +84,21 @@ export const Programs: React.FC<ProgramsProps> = (props: ProgramsProps) => {
         const title = values[titleColumnIndex]
         const summary = values[summaryColumnIndex]
         const link = values[linkColumnIndex]
-        const iconValue = values[iconColumnIndex]
+        const iconFileHandleIdValue = values[iconColumnIndex]
         const matchValue = values[matchColIndex]
 
+        const fileHandleAssociation = getFileHandleAssociation(
+          entity.data,
+          iconFileHandleIdValue,
+          `syn${el.rowId}`,
+        )
```

Comment:
Not sure if inline calls to getFileHandleAssociation need to be memoized. My concern is that each render will create an new fileHandleAssociation object, causing ImageFileHandle to re-render and re-fetch the image.

If you didn't see this happen locally, then we probably don't need to worry about it.